src/engine/engine.ts

```diff
@@ -75,7 +75,7 @@ function extractOnlyNPMOptions({ access, tag, otp, dryRun }: Schema) {
   };
 }
 
-function prepareOptions(origOptions: Schema, logger: logging.LoggerApi) {
+function prepareOptions(origOptions: Schema, logger: logging.LoggerApi): Schema {
```

Comment:
Thanks the implicit returning type of typescript is already returning the `Schema` type but I think is better to put it explicitly like you did, thanks for making this 🙌